exercises/practice/hamming/.meta/config.json

```diff
@@ -1,6 +1,16 @@
 {
   "blurb": "Calculate the Hamming difference between two DNA strands.",
   "authors": [],
```

Comment:
author not on github

https://github.com/exercism/csharp/commit/23b6b434a744f7b00c9dafbba593d6b16668da63